test/image-blueprints-bootc/layer2-source/group1/microshift-ovsdb-ownership.conf.template

```diff
@@ -0,0 +1,2 @@
+[Service]
+ExecStartPre=/bin/chown -R openvswitch. /etc/openvswitch
```

Comment:
Any `chown -R` without `-h` is almost always a bug

Reply:
What's up with the `.`? Do you mean `:`?

Reply:
The `username<dot>` syntax means the ownership is changed for the default group of the user.

Reply:
Added -h option.